app/db_placeholder.go

```diff
@@ -0,0 +1,27 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+
+//go:build !rocksdb
+// +build !rocksdb
+
+package app
+
+import (
+	"errors"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	storetypes "github.com/cosmos/cosmos-sdk/store/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+const versionDB = "versiondb"
```

Comment:
already defined?

Reply:
we have to include it in both files, `db.go` & `db_placeholder.go` cause only one of these is compiled depending on the build (with or without rocksdb)